activejob/test/integration/queuing_test.rb

```diff
@@ -45,6 +45,13 @@ class QueuingTest < ActiveSupport::TestCase
     end
   end
 
+  test "should supply a wrapped class name to DelayedJob" do
+    skip unless adapter_is?(:delayed_job)
+    ::HelloJob.perform_later
+    job = Delayed::Job.first
+    assert job.name =~ /HelloJob \[[0-9a-f-]+\] from DelayedJob\(default\) with arguments: \[\]/
```

Comment:
Could you used `assert_match`?

Reply:
@rafaelfranca 
Sure, I've just changed it. 👍